libs/overlay/src/lib/ChatOverlay.ts

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
Is this really working? This is react specific thing or I'm wrong?

Reply:
@mikitabut , it's NextJS feature

Reply:
This ChatOverlay.ts can be loaded outside of NextJS, so we shouldn't use it here

Reply:
@IlyaBondar looks like its react :)
https://react.dev/reference/rsc/use-client

Reply:
[This is Next JS feature](https://nextjs.org/docs/app/building-your-application/rendering/client-components#using-client-components-in-nextjs)